GliaWidgetsObjC/README.md

```diff
@@ -0,0 +1,23 @@
+## Objective-C wrapper for [GliaWidgets](https://github.com/salemove/ios-sdk-widgets)
```

Comment:
I think that tech writers need to go through this Readme to ensure that the language is appropriate.